CODEOWNERS

```diff
@@ -51,7 +51,6 @@ components/StarSearch/SuggestedPrompts.tsx @zeucapua @nickytonline @brandonrober
 components/TabList/tab-list.tsx @zeucapua @jpmcb @brandonroberts
 components/Tables/ContributorsGrid.tsx @zeucapua @brandonroberts @nickytonline
 components/Tables/ContributorsTable.tsx @zeucapua @zeucapua @brandonroberts
-components/Workspaces/InsightUpgradeModal.tsx @brandonroberts
```

Comment:
Does the pizza-cli handle removing attributions for removed files @jpmcb? If so, I'll undo this change.

Reply:
It should but this change is fine: feel enabled to make codeowner changes for files you're changing!